doc/src/manual/metaprogramming.md

```diff
@@ -419,9 +420,9 @@ this is an extremely useful tool for debugging macros):
 
 ```julia
 julia> ex = macroexpand( :(@sayhello("human")) )
```

Comment:
And could be doctested if the note on the interpolated string is extracted and added afterwards instead